src/qrcode/components/QRTabBar.tsx

```diff
@@ -91,33 +77,14 @@ export default function QRTabBar({
   return (
     <SafeAreaView style={styles.container} edges={['top']}>
       <View style={styles.leftContainer}>
-        <TopBarIconButton
-          icon={leftIcon === 'times' ? <Times color={color} /> : <BackChevron color={color} />}
-          onPress={onPressClose}
-        />
+        <TopBarIconButton icon={<Times color={color} />} onPress={onPressClose} />
       </View>
-      {canSwitch ? (
```

Comment:
If we can't switch then we should just not use the QRNavigator or the QRTabBar. Opted to instead create the `ScanQRScreen` to handle the case when the user shouldn't be able to switch